packages/twenty-server/src/engine/api/graphql/graphql-query-runner/services/api-event-emitter.service.ts

```diff
@@ -46,20 +54,28 @@ export class ApiEventEmitterService {
     );
 
     this.workspaceEventEmitter.emit(
-      `${objectMetadataItem.nameSingular}.updated`,
+      `${objectMetadataItem.nameSingular}.${DatabaseEventAction.UPDATED}`,
       records.map((record) => {
+        const before = this.removeGraphQLAndNestedProperties(
+          mappedExistingRecords[record.id],
+        );
+        const after = this.removeGraphQLAndNestedProperties(record);
+        const diff = objectRecordChangedValues(
+          before,
+          after,
+          updatedFields,
+          objectMetadataItem,
+        );
+
         return {
           userId: authContext.user?.id,
           recordId: record.id,
           objectMetadata: objectMetadataItem,
           properties: {
-            before: mappedExistingRecords[record.id]
-              ? this.removeGraphQLAndNestedProperties(
-                  mappedExistingRecords[record.id],
-                )
-              : undefined,
-            after: this.removeGraphQLAndNestedProperties(record),
+            before,
+            after,
             updatedFields,
+            diff,
```

Comment:
ok why not adding the diff to the event api

Reply:
what do you mean by "event api"?